src/Updater/UpdaterManager.php

```diff
@@ -52,7 +52,7 @@ public function provider($name = null)
      */
     public function driver($driver = null)
     {
-        return $this->store($driver);
+        return $this->resolve($driver);
```

Comment:
Can you explain this change?

Reply:
Easy. `store` never existed 

I did searched through git history 😄